java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -818,6 +818,13 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
         final String consumer1 = "{consumer}-1-" + UUID.randomUUID();
         final String streamKey2 = "{streamKey}-2-" + UUID.randomUUID();
         final String groupName3 = "{groupName}-2-" + UUID.randomUUID();
+        final HashMap xinfoGroupsExpected = new HashMap();
```

Comment:
Add type parameters

Reply:
And test glidestring variant of the method too

Reply:
No glidestring tests have been added I assume because the newBuildArgs converts already. Just following set convention by AWS.

Reply:
Ok, add type parameters to the map please